Classes/Domain/Model/Order/Tax.php

```diff
@@ -16,19 +16,9 @@
 
 class Tax extends AbstractEntity
 {
-    #[Validate(['validator' => 'NotEmpty'])]
-    protected float $tax;
-
-    #[Validate(['validator' => 'NotEmpty'])]
-    protected TaxClass $taxClass;
-
-    public function __construct(
-        float $tax,
-        TaxClass $taxClass
-    ) {
-        $this->tax = $tax;
-        $this->taxClass = $taxClass;
-    }
+    public function __construct(#[Validate(['validator' => 'NotEmpty'])]
+        protected float $tax, #[Validate(['validator' => 'NotEmpty'])]
+        protected TaxClass $taxClass) {}
```

Comment:
Maybe the attributes should be in the same line as the argument they belong to?

Reply:
Fix this with e91a033.